flink-connectors/flink-connector-elasticsearch6/src/test/java/org/apache/flink/streaming/connectors/elasticsearch/table/Elasticsearch6DynamicSinkITCase.java

```diff
@@ -61,7 +65,20 @@
 public class Elasticsearch6DynamicSinkITCase {
 
 	@ClassRule
-	public static ElasticsearchResource elasticsearchResource = new ElasticsearchResource("es-6-dynamic-sink-tests");
+	public static ElasticsearchContainer elasticsearchContainer =
```

Comment:
nit: the linebreaks look weird (ditto on other container fields).

Reply:
It's the Google Java Style, which is a subset (or superset depending on where you're coming from 😅) of our (very relaxed) coding style so I prefer that one.